doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -51,7 +51,10 @@ See `Samples`_ for lists of changes for the protocol-related samples.
 Bluetooth LE
 ------------
 
-|no_changes_yet_note|
+* Added:
+
+  * Support for hci_driver_close, so bt_disable can now be used to disable the sdc.
+  * The Kconfig option :kconfig:option:`CONFIG_BT_UNINIT_MPSL_ON_DISABLE` was added so that mpsl is uninitialized with bt_disable, which release all peripherals used by MPSL.
```

Comment:
Minor changelog sugestion:
```suggestion
  * Support for :c:func:`hci_driver_close`, so :c:func:`bt_disable` can now be used to disable the SoftDevice Controller.
  * The Kconfig option :kconfig:option:`CONFIG_BT_UNINIT_MPSL_ON_DISABLE` was added so that MPSL is uninitialized with :c:func:`bt_disable`, which releases all peripherals used by MPSL.
```

Reply:
These are good changes. Just want to further improve the second bullet:
```suggestion
  * Support for :c:func:`hci_driver_close`, so :c:func:`bt_disable` can now be used to disable the SoftDevice Controller.
  * The Kconfig option :kconfig:option:`CONFIG_BT_UNINIT_MPSL_ON_DISABLE` that, when enabled, uninitializes the MPSL when :c:func:`bt_disable` is used.
    This releases all peripherals used by the MPSL.
```
I believe the meaning is still the same.